pkg/server/http.go

```diff
@@ -48,6 +52,7 @@ var DefaultConfig = Config{
 	Addr:             "localhost",
 	ServerHeader:     "Overlay API",
 	AdminBearerToken: uuid.NewString(),
+	ARC:              "",
```

Comment:
IMHO it would be better to have some default 
I don't know what it's exactly is, but 
1. if this is token for ARC it could be the same token as we use in spv-wallet - or we can ask Deggen for some dedicated token for overlays.
2. if this is token for ARC callback endpoint - then it could be some static string